app/jobs/channelJobs.js

```diff
@@ -704,8 +710,52 @@ var job = {
             return { success: 0, message: ex };
         }
     },
-   
 
+    sentMailForOpenTickets: async function (config) {
+        try {
+            var emailConfig = {};
+            var jira = {
+                status: "Open",
+                page: "All"
+            }
+            var openTickets = await this.getAllJira(jira);
+            if (config.openJiraTicketConfig.channels.indexOf('Email')) {
+                emailConfig.SMTPConfig = await this.getSMTPConfig();
+                emailConfig.createdBy = openTickets.issues.reduce(function (r, a) {
+                    r[a.createdBy] = r[a.createdBy] || [];
+                    r[a.createdBy].push(a);
+                    return r;
+                }, Object.create(null));
+
+                emailConfig.reporter = openTickets.issues.reduce(function (r, a) {
+                    r[a.reporter] = r[a.reporter] || [];
+                    r[a.reporter].push(a);
+                    return r;
+                }, Object.create(null));
+
+                emailConfig.assign = openTickets.issues.reduce(function (r, a) {
+                    r[a.assignPerson] = r[a.assignPerson] || [];
+                    r[a.assignPerson].push(a);
+                    return r;
+                }, Object.create(null));
+                modelsUtil.sendNotification(emailConfig);
+            }
+            if (config.openJiraTicketConfig.channels.indexOf('Teama')) {
```

Comment:
rename it to Team...try to use constant instead...better to implement command line and separate function for if and else processing